### PR TITLE
chore: update nugets in .Tests and .TestingHarnessTest

### PR DESCRIPTION
**Pull request by maw136**
Chore, update nugets in Tests and TestHarness.
Should have no impact on main dependencies of generated nugets. 
Added necessary changes after nuget update

Fixes https://github.com/microsoft/playwright-dotnet/issues/2557

### CODE REVIEW

File: src/Playwright/Core/WritableStream.cs [hunk 1]
```diff
@@ -53,42 +53,51 @@ internal WritableStream(IChannelOwner parent, string guid) : base(parent, guid)
...
+        {
+            _stream.CloseAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+        }
 
-    public override long Seek(long offset, SeekOrigin origin) => throw new NotImplementedException();
+        base.Dispose(disposing);
+    }
 
-    public override void SetLength(long value) => throw new NotImplementedException();
+    public override long Seek(long offset, SeekOrigin origin) => throw new NotSupportedException();
```
Comment (mxschmitt) [inline src/Playwright/Core/WritableStream.cs:94]: are all these C# changes caused by bumping Roslynator or how to they relate to the version bumps?

Comment (maw136) [reply src/Playwright/Core/WritableStream.cs:94]: These `Stream` based changes are strictly doc-based - no Roslynator detection there, maybe some async but I don't remember. I have posted link to `Stream` docs in commit description.

Comment (maw136) [reply src/Playwright/Core/WritableStream.cs]: Extracted to separate PR

**Comment by maw136** [comment]
Depends on:
https://github.com/microsoft/playwright-dotnet/pull/2593
https://github.com/microsoft/playwright-dotnet/pull/2591

**Comment by mxschmitt** [comment]
Closing for now since its stale.

**Pull request closed without merging**